### ISSUE TITLE
"chromeWebSecurity": false not reflected in Electron

### ISSUE THREAD

**Issue by keeema** [cypress-io/cypress#8412]
### Current behavior:

Send requests using XMLHttpRequest  not respecting CORS policy.
Have configuration `"chromeWebSecurity": false,`

Chrome windowed and headless:
No error

Electron windowed and headless:
Access to XMLHttpRequest at 'url1' from origin 'url2' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

### Desired behavior:

No error in Electron. (Same as in Chrome 84 - there it is OK)

### Versions

Cypress 5.0.0
Electron 83

**Comment by jennifer-shehane** [comment]
Unfortunately we/'' have to close this issue if there is not enough information to reproduce the problem. This does not mean that your issue is not happening - it just means that we do not have a path to move forward. 

Please comment in this issue with a reproducible example so that we can investigate.

**Comment by flotwig** [comment]
Learned some more about this issue, it appears to be tied to an upstream bug in Electron indeed: electron/electron#23664

Looks like this is a duplicate of #8399, so I'll close this. @keeema I've updated the other issue #8399 with some more info and a potential workaround.
